### DISCUSSION TITLE
Loading 이미지 투표

### DISCUSSION THREAD

**Discussion by NamgyungKim** [prgrms-fe-devcourse/FEDC2_2JAVATAYO_Dongyoung#200] [Ideas]
페이지 로딩중을 만드려고하는데... 

![animation_640_l4lc1bj6](https://user-images.githubusercontent.com/87519250/174483411-72a3991e-73ed-4d1e-a976-20f8834671f6.gif)

를 기반으로 .. 아래와같은 두가지가나왔습니다. 

![Loading](https://user-images.githubusercontent.com/87519250/174483443-560748fd-bf6e-47fb-bfee-08da801cc5b7.jpg)

- [ ] 왼쪽
- [x] 오른쪽
- [ ] 그외...

**Comment by Hyevvy** [comment]
와 오른쪽 예쁘네용 ㄷ ㄷ
금손 ... 😳

**Comment by hyunjin0910** [comment]
전 오른쪽에 한표 👋

**Comment by YuHyun-P** [comment]
저도 오른쪽에 한 표ㅎㅎ

**Comment by hwoo3303** [comment]
저도 오른쪽에 한표요!!
